src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -1586,16 +1586,22 @@ private class EventReceiver : IDisposable
             private readonly object server;
             private readonly EventInfo eventInfo;
             private readonly Delegate registeredHandler;
+            private readonly string rpcEventName;
 
-            internal EventReceiver(JsonRpc jsonRpc, object server, EventInfo eventInfo)
+            internal EventReceiver(JsonRpc jsonRpc, object server, EventInfo eventInfo, JsonRpcTargetOptions options)
             {
                 Requires.NotNull(jsonRpc, nameof(jsonRpc));
                 Requires.NotNull(server, nameof(server));
                 Requires.NotNull(eventInfo, nameof(eventInfo));
 
+                options = options ?? JsonRpcTargetOptions.Default;
+
                 this.jsonRpc = jsonRpc;
                 this.server = server;
                 this.eventInfo = eventInfo;
+
+                this.rpcEventName = options.MethodNameTransform != null ? options.MethodNameTransform(eventInfo.Name) : eventInfo.Name;
```

Comment:
Are we confident that having a single transform for method names and event names is appropriate? The alternative I'm thinking about is offering a second transform function property so folks can have one policy for events and one for methods. Since they are implemented by different parties, and the tense of the method name may be different, I wonder if it might be useful at some point.

Reply:
Resolved